src/web-ui/src/analytics/AnalyticsHandler.js

```diff
@@ -514,5 +542,24 @@ export const AnalyticsHandler = {
 
     amplitudeEnabled() {
         return process.env.VUE_APP_AMPLITUDE_API_KEY && process.env.VUE_APP_AMPLITUDE_API_KEY != 'NONE'
-    }
+    },
+
+    optimizelyEnabled() {
+        return !!process.env.OPTIMIZELY_SDK_KEY;
```

Comment:
The SSM parameters used to populate the environment variables for the web app are initially created with a value of `NONE` (they can't be blank). So checking for `!= 'NONE'`  is important for that case too.